### ISSUE TITLE
Eosinophilic pneumonia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6248]
Is this a synonym for Pulmonary eosinophilic infiltration HP:0032071

https://hpo.jax.org/app/browse/term/HP:0032071

It is used to describe this disease https://pubmed.ncbi.nlm.nih.gov/27427983/

**Comment by GrieseAG** [comment]
I would use "Eosinophilic pneumonia" as **Preferred term label** of HP:0032071

**Definition:** _Comment_: Pulmonary function testing usually reveals a restrictive process with reduced diffusion capacity for carbon monoxide. Laboratory tests typical include increased levels of eosinophils in the blood, a high erythrocyte sedimentation rate, iron deficiency anemia, and increased platelets. 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**   eosinophilic granulomatosis with polyangiitis

**Comment by pnrobinson** [comment]
done as suggested, thanks
